independent-publisher-connectors/Bluesky Social/apiProperties.json

```diff
@@ -0,0 +1,73 @@
+{
```

Comment:
Please update apiProerities.json file as per the below template and schema. 

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiProperties.json

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/schemas/paconn-apiProperties.schema.json

Reply:
@vmanoharas I've made updates here and will re-review to make sure they jive with the standards provided in the links. Thank you!